cypress/features/regression/test/flatTableNoiFrame.feature

```diff
@@ -2,7 +2,7 @@ Feature: FlatTable component
   I want to check FlatTable component properties
 
   Background: Open FlatTable component page in no iFrame
-    Given I open Test "Flat Table" component basic page with prop value
+    Given I open Design System Flat Table Test component basic page with prop value
```

Comment:
```suggestion
    Given I open "Design System Flat Table Test" component page "basic" with prop value
```

Reply:
Good spot.

This one is actually different to the others, they're not arguments because it calls a specific story in `visitFlatTableComponentNoiFrame`.